dbms/src/Server/Server.cpp

```diff
@@ -178,27 +178,33 @@ class TCPConnectionFactory : public Poco::Net::TCPServerConnectionFactory
 };
 
 
-int Server::main(const std::vector<std::string> & args)
+static std::string getCanonicalPath(std::string && path)
 {
-	Logger * log = &logger();
-
-	std::string path = config().getString("path");
 	Poco::trimInPlace(path);
 	if (path.empty())
 		throw Exception("path configuration parameter is empty");
 	if (path.back() != '/')
 		path += '/';
+	return path;
+}
+
+
+int Server::main(const std::vector<std::string> & args)
+{
+	Logger * log = &logger();
```

Comment:
The same line is in [master](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Server/Server.cpp#L183) now. It is used in `Server::main` for [logging](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Server/Server.cpp#L235).